keps/sig-api-machinery/4020-unknown-version-interoperability-proxy/README.md

```diff
@@ -871,7 +851,15 @@ Focusing mostly on:
     heartbeats, leader election, etc.)
 -->
 
-No.
+Yes, enabling this feature will result in new API calls.  Specifically:
+
+- Discovery calls via a loopback client: The local apiserver will use a loopback
+client to discover the resources it serves for each group-version. This should
+only happen once upon server startup.
+- Remote discovery calls to peer apiservers: The event handler for apiserver identity
+lease informer will make remote discovery calls to each peer apiserver whose
+lease object is added or updated. This is expected to happen everytime an
+apiserver identity lease is created or updated.
```

Comment:
Is updated correct? Do we need to refresh on lease renewal?

Reply:
Modified to say "updated as a result of change in [holderIdentity](https://github.com/kubernetes/kubernetes/blob/release-1.32/staging/src/k8s.io/api/coordination/v1/types.go#L58) implying a server restart"